custom-vu/src/main/kotlin/jces1209/vu/page/IssueNavigator.kt

```diff
@@ -1,17 +1,52 @@
 package jces1209.vu.page
 
 import com.atlassian.performance.tools.jiraactions.api.WebJira
+import jces1209.vu.page.ViewSubscriptions.ViewSubscriptions
+import jces1209.vu.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.support.ui.ExpectedConditions
+import org.openqa.selenium.support.ui.ExpectedConditions.*
 
 abstract class IssueNavigator(
-    private val jira: WebJira
+    protected val jira: WebJira
 ) {
     protected val driver = jira.driver
-    
+    protected val filterSubscriptionList = and(
+        presenceOfElementLocated(By.id("filter-subscription")),
+        presenceOfElementLocated(By.id("filter.subscription.prefix.daysOfMonth")),
+        presenceOfElementLocated(By.xpath("//*[. = 'Filter Subscription']")),
+        presenceOfElementLocated(By.xpath("//*[. = 'Recipients']")),
+        presenceOfElementLocated(By.xpath("//*[. = 'Schedule']")),
+        presenceOfElementLocated(By.xpath("//*[. = 'Interval']")),
+        presenceOfElementLocated(By.id("filter-subscription-submit")),
+        presenceOfElementLocated(By.id("filter-subscription-cancel"))
+    )
+
     abstract fun waitForNavigator()
     abstract fun selectIssue()
+    abstract fun clickDetails()
+    abstract fun clickNewSubscription()
+    abstract fun subscribe()
+    abstract fun manageSubscriptions(): ViewSubscriptions
 
     fun openNavigator(): IssueNavigator {
         jira.goToIssueNavigator("resolution = Unresolved ORDER BY priority DESC")
         return this;
     }
+
+    fun checkSubscriptions() {
+        val detailsView = driver.wait(
+            visibilityOfElementLocated(By.id("inline-dialog-filter-details-overlay")))
+        driver.wait(
+            ExpectedConditions.and(
+                visibilityOfNestedElementsLocatedBy(detailsView, By.className("search-owner")),
+                visibilityOfNestedElementsLocatedBy(detailsView, By.xpath("//*[contains(text(), 'Owned by')]")),
```

Comment:
locators, which are related to text, doesn't look like stable
```suggestion
                visibilityOfNestedElementsLocatedBy(detailsView, By.className("search-owner-name")),
```